### ISSUE TITLE
PHP 5.4 requirement

### ISSUE THREAD

**Issue by pftaylor61** [afragen/github-updater#401]
My servers (Webhosting.uk.com) do not currently have plans to upgrade from PHP 5.3. Therefore, your new updates to Github-Updater will not work. I guess I could create a fork, for my own purposes, but thought you should be aware of the problem.

**Comment by afragen** [comment]
OK, I fixed the code to allow for PHP 5.3 https://github.com/afragen/github-updater/commit/ffb6a1b37542869653148356acc9b0ef92e76e8e

But seriously, PHP 5.4 is not even supported any longer. You should either push your host to update or move. They put you and your clients at risk by not supporting current, supported versions of PHP. 

Siteground and DreamHost are both very good.

**Comment by pftaylor61** [comment]
I have successfully updated to PHP 5.6. My hosts are not using anything
between 5.6 and 7.0. When I tried 7.0, 2 of my sites failed, but the other
4 worked - so I guess I need to check the themes and plugins for the 2 that
failed. Meanwhile, I am going to try and work with 5.6, so I should be able
to use your new Github updater version. I use your plugin a lot, because
all of my own plugins and themes are being updated by it.

Paul


*Paul Taylor, BSc MEd*
850-450-2310
pftaylor61@gmail.com
Creation Speaker
Author of "Don't Miss the Boat" and "The Six Days of Genesis"
